x/pairing/keeper/epoch_payments.go

```diff
@@ -86,7 +86,17 @@ func (k Keeper) AddEpochPayment(ctx sdk.Context, chainID string, epoch uint64, u
 	if !found {
```

Comment:
We've checked and saw that the pointer do change. After CNS-262 will be merged (changing the payment objects to hold keys rather than pointers), I will address this issue